src/lib/jest-globals.js

```diff
@@ -0,0 +1,74 @@
+function notImplemented() {
+	throw Error(`Not Implemented`);
+}
+
+// @todo expect.extend() et al
+
+global.jest = {
+	addMatchers(matchers) {
+		jasmine.addMatchers(matchers);
+	},
+	advanceTimersByTime(msToRun) {
+		// _getFakeTimers().advanceTimersByTime(msToRun);
+		notImplemented();
+	},
+	advanceTimersToNextTimer(steps) {
+		// _getFakeTimers().advanceTimersToNextTimer(steps);
+		notImplemented();
+	},
+	autoMockOff: notImplemented,
+	autoMockOn: notImplemented,
+	clearAllMocks: notImplemented,
+	clearAllTimers() {
+		// _getFakeTimers().clearAllTimers();
+		notImplemented();
+	},
+	createMockFromModule(moduleName) {
+		// return this._generateMock(from, moduleName);
+		notImplemented();
+	},
+	deepUnmock: notImplemented,
+	disableAutomock: notImplemented,
+	doMock: notImplemented,
+	dontMock: notImplemented,
+	enableAutomock: notImplemented,
+	fn: jasmine.createSpy,
```

Comment:
Looks like we can also use `jest-mock` to implement some of these mocking functions. Probably can't do the module mocking functions but for the other core mocking functions, `jest-mock` doesn't import/require any external modules. I ran its test suite in the browser and it seems to work! So it may not be hard to bring over some of the core mocking functions directly from jest.

Tests I checked: https://github.com/facebook/jest/blob/e8b7f57e05e3c785c18a91556dcbc7212826a573/packages/jest-mock/src/__tests__/index.test.ts#L12-L1399
(do have to remove some TypeScript on lines 13-15 and 228-229

Fake test harness I used to run the tests (which require some node stuff even though the actual implementation does not) (leaving this here for myself in case I want to look at this again):
```js
var s = document.createElement('script');
s.src = 'https://unpkg.com/expect@25.5.0/build-es5/index.js';
document.body.appendChild(s);

window.global = window;
window.module = { exports: {} };
var s = document.createElement('script');
s.src = 'https://unpkg.com/jest-mock@26.3.0/build/index.js';
s.onload = () => {
	setupModuleMocker();
	console.log(moduleMocker);
};
document.body.appendChild(s);

function setupModuleMocker() {
	let ModuleMocker = module.exports.ModuleMocker;
	let moduleMocker = new ModuleMocker(window);

	window.ModuleMocker = ModuleMocker;
	window.moduleMocker = moduleMocker;
	window.jest = {
		fn: moduleMocker.fn.bind(moduleMocker),
	};
}

window.describe = (name, fn) => {
	fn();
};

let beforeEaches = [];
window.beforeEach = (fn) => {
	beforeEaches.push(fn);
};

let testCount = { total: 0, success: 0, failed: 0 };
window.it = window.test = (name, fn) => {
	setupModuleMocker();
	beforeEaches.forEach((fn) => fn());
	try {
		testCount.total++;
		fn();
	} catch (e) {
		testCount.failed++;
		console.error(e);
		console.log(`FAILED: ${name}`);
		return;
	}

	testCount.success++;
	console.log(`SUCCESS: ${name}`);
};

window.vm = {
	createContext() {
		return window;
	},
	runInNewContext(code, context) {
		var f = new Function('runInNewContext_Code', `return eval(\`${code}\`)`);
		return f.call(context);
	},
	runInContext(code, context) {
		return eval(code);
	},
};
```